### PR TITLE
📖 Update contributing guide

### PR DESCRIPTION
**Pull request by zhujian7**
## Related issue(s)

Fixes #

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,40 +1,24 @@
...
-| work | staging |
-| registration-operator | staging |
-
-Status:
-* staging: the component is having all code in `staging` folder.
-* mix: the component is having code in both `staging` and outside.
-* done: the component is having all code outside `staging` folder and done of the migration.
-
-If a component is in the `staging` status, you need to open project in the `/staging` folder and contribute there.
+Before 0.11.0, the 5 components has independent repos, now we are going through [the task of consolidating code](https://github.com/open-cluster-management-io/OCM/issues/128) to merge all code into this repo to gain all kinds of benifits.
```
Comment (qiujian16) [inline CONTRIBUTING.md:21]: we have consolidate the code into ocm repo.

The task is done, so we should not it is going through.

Comment (zhujian7) [reply CONTRIBUTING.md]: removed this line

**Comment by xuezhaojun** [comment]
👍lgtm

**Comment by qiujian16** [comment]
do we need to apply this for all repos? do we also need to add a start dev enviornment section in this or in the website?

**Comment by zhujian7** [comment]
@qiujian16 @xuezhaojun PTAL

**Comment by lenicatko** [comment]
> do we need to apply this for all repos? do we also need to add a start dev enviornment section in this or in the website?

As a new comer to OCM, I noticed that each repo has its own contributing guide, but they are mostly the same. If it make sense and each repo requires something different, it is ok. But if not, it would be good to consolidate it. 

You have also a community repo, where you updated a contributor ladder, but it is not linked in any other repo. Try to focus on that repo, and make sure you make reference to it in each repo, so you don't need to update everything multiple times.

**Comment by xuezhaojun** [comment]
lgtm to me

**Comment by qiujian16** [comment]
/hold

should we move this to the community repo and just put a link here?

**Comment by zhujian7** [comment]
@qiujian16 I think we can merge this into the ocm repo, since different repos have different developing process, (e.g API, addon-framework, clusteradm).

**Pull request merged**